### PR TITLE
Alpha sort keys in Cargo.toml files

### PR DESCRIPTION
**Pull request by jayvdb**
Followup to https://github.com/Electron100/butane/pull/39, this uses `reorder_keys`.  No worries if you reject because it also changes the package metadata sections.
If rejected, I'll see if I can find another tool which can reorder only the dependencies.

### CODE REVIEW

**Comment by Electron100** [comment]
Hmm, I'm going to have to think about this one a little bit. The ordering of dependencies is nice (I've been trying to keep them ordered manually, but obviously I haven't fully succeeded), but I don't love the re-ordering of other sections.

File: butane/Cargo.toml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 
 [features]
-default = ["datetime", "json", "uuid"]
-json = ["butane_core/json"]
-sqlite = ["butane_core/sqlite"]
-sqlite-bundled = ["butane_core/sqlite-bundled"]
-pg = ["butane_core/pg"]
 datetime = ["butane_core/datetime", "butane_codegen/datetime"]
 debug = ["butane_core/debug"]
+default = ["datetime", "json", "uuid"]
```
Comment (jayvdb) [inline butane/Cargo.toml:17]: putting this in the middle is also a significant degradation IMO.  I suspect most Rust developers look at Cargo.toml of libraries they are using, and the default feature list is very commonly what they are looking for.

I've created a bunch of issues like this https://github.com/tbrand/cargo-tomlfmt/issues/25 , as that tool is more likely to support Cargo.toml specific formatting than the more general `taplo`

Comment (jayvdb) [reply butane/Cargo.toml:17]: More tools at https://github.com/TimonPost/cargo-toml-format#similar-work .  I'll try them out and see what works best.

**Comment by jayvdb** [comment]
I've updated this.  I havent built CI to enforce it yet - there are only a few contributors atm, so not a high priority atm.

**Review by Electron100 — APPROVED** [review]

**Pull request merged**